sdk/node/Libplanet.Node/Options/SeedOptions.cs

```diff
@@ -13,6 +13,10 @@ public sealed class SeedOptions : OptionsBase<SeedOptions>
     [Description("The private key of the seed node.")]
     public string PrivateKey { get; set; } = string.Empty;
 
+    [AppProtocolVersion]
+    [Description("The version of the application protocol.")]
+    public string AppProtocolVersion { get; set; } = string.Empty;
+
```

Comment:
`Swarm`, `Validator`, and `Seed` options have a common members like `APV`, and `PrivateKey`. How about using inherit?

Reply:
There is an issue in the schema where the description of the option cannot be different for each option.

Reply:
Added `NodeOptionsBase` for common properties [[link](https://github.com/planetarium/libplanet/pull/3932/commits/dbdac35fe5cf5bd56ed04a4f37522da312903591#diff-3e642b7fb403318879dc8fd90dfe2afc67c2571293ba5781f9bada6e04e50eff)]